Engine/Results/LiveTradingResultHandler.cs

```diff
@@ -697,13 +697,31 @@ public void SampleRange(List<Chart> updates)
                     foreach (var series in update.Series.Values)
                     {
                         //If we don't already have this record, its the first packet
-                        if (!Charts[update.Name].Series.ContainsKey(series.Name))
+                        var chart = Charts[update.Name];
+                        if (!chart.Series.ContainsKey(series.Name))
                         {
-                            Charts[update.Name].Series.Add(series.Name, new Series(series.Name, series.SeriesType, series.Index, series.Unit));
+                            chart.Series.Add(series.Name, new Series(series.Name, series.SeriesType, series.Index, series.Unit)
+                            {
+                                Color = series.Color, ScatterMarkerSymbol = series.ScatterMarkerSymbol
```

Comment:
Added this here to match backtesting -- @jaredbroad -- please confirm that this wasn't intentionally left off of here.

Reply:
Nice catch ty